src/verilog.y

```diff
@@ -5230,87 +5238,83 @@ endLabelE<strp>:
+        |       yDEFAULT yCLOCKING clocking_event ';' clocking_itemListE yENDCLOCKING endLabelE
+                        { $$ = new AstClocking{$<fl>2, "", $3, $5, true}; }
+        |       yDEFAULT yCLOCKING idAny clocking_event ';' clocking_itemListE yENDCLOCKING endLabelE
+                        { $$ = new AstClocking{$<fl>3, *$3, $4, $6, true}; }
+//UNSUP |       yGLOBAL__CLOCKING yCLOCKING clocking_event ';' clocking_itemListE yENDCLOCKING endLabelE
+//UNSUP                 { $$ = new AstClocking{$<fl>1, "", $2, $4, /*global*/}; }
+//UNSUP |       yGLOBAL__CLOCKING yCLOCKING idAny clocking_event ';' clocking_itemListE yENDCLOCKING endLabelE
+//UNSUP                 { $$ = new AstClocking{$<fl>2, *$2, $3, $5, /*global*/}; }
+        ;
 
-//UNSUPclocking_itemList:  // IEEE: [ clocking_item ]
-//UNSUP         clocking_item                           { $$ = $1; }
-//UNSUP |       clocking_itemList clocking_item         { $$ = addNextNull($1, $2); }
-//UNSUP ;
+clocking_event<senItemp>:       // IEEE: clocking_event
+                '@' id
+                        { $$ = new AstSenItem{$<fl>2, VEdgeType::ET_CHANGED, new AstParseRef{$<fl>2, VParseRefExp::PX_TEXT, *$2, nullptr, nullptr}}; }
+        |       '@' '(' event_expression ')'            { $$ = $3; }
+        ;
 
-//UNSUPclocking_item:  // ==IEEE: clocking_item
-//UNSUP         yDEFAULT default_skew ';'               { }
-//UNSUP |       clocking_direction list_of_clocking_decl_assign ';'     { }
+clocking_itemListE<clockingItemp>:
+                /* empty */                             { $$ = nullptr; }
+        |       clocking_itemList                       { $$ = $1; }
+        ;
+
+clocking_itemList<clockingItemp>:  // IEEE: [ clocking_item ]
+                clocking_item                           { $$ = $1; }
+        |       clocking_itemList clocking_item         { $$ = addNextNull($1, $2); }
+        ;
+
+clocking_item<clockingItemp>:   // IEEE: clocking_item
+                yDEFAULT yINPUT clocking_skew ';'       { $$ = new AstClockingItem{$<fl>1, VDirection::INPUT, $3, nullptr}; }
+        |       yDEFAULT yOUTPUT clocking_skew ';'      { $$ = new AstClockingItem{$<fl>1, VDirection::OUTPUT, $3, nullptr}; }
+        |       yDEFAULT yINPUT clocking_skew yOUTPUT clocking_skew ';'
+                        { $$ = new AstClockingItem{$<fl>1, VDirection::INPUT, $3, nullptr};
+                          $$->addNext(new AstClockingItem{$<fl>4, VDirection::OUTPUT, $5, nullptr}); }
+        |       yINPUT clocking_skewE list_of_clocking_decl_assign ';'
+                        { $$ = new AstClockingItem{$<fl>1, VDirection::INPUT, $2, $3}; }
+        |       yOUTPUT clocking_skewE list_of_clocking_decl_assign ';'
+                        { $$ = new AstClockingItem{$<fl>1, VDirection::OUTPUT, $2, $3}; }
+//UNSUP |       yINPUT clocking_skewE yOUTPUT clocking_skewE list_of_clocking_decl_assign ';' { }
+//UNSUP |       yINOUT list_of_clocking_decl_assign ';' { }
```

Comment:
Ideally these would give unsupported errors, so the user knows it is legal syntax versus not supported.